### PR TITLE
cmd: add "notes" column in list and find outputs; add developer to find output; move price in find output into notes

### PR DESCRIPTION
**Pull request by chipaca**
This builds on `local-devmode-in-daemon` (#1224) so [compare with that](https://github.com/chipaca/snappy/compare/local-devmode-in-daemon...chipaca:notes-in-list).

### CODE REVIEW

File: cmd/snap/notes.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+package main
+
+import (
+	"strings"
+
+	"github.com/snapcore/snapd/client"
+)
+
+type Notes struct {
```
Comment (niemeyer) [inline cmd/snap/notes.go]: Very clean approach, thanks!

**Comment by niemeyer** [comment]
LGTM, but needs to wait on the resolutions of the pre-req.

**Comment by mvo5** [comment]
Very nice indeed! :+1:

**Comment by mvo5** [comment]
retest this please

**Comment by zyga** [comment]
Just adding my 0.01 +1 :-)

**Comment by niemeyer** [comment]
retest this please

**Pull request merged**